<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Angle Tolerance              </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QDoubleSpinBox" name="AngleTolerance" >
       <property name="layoutDirection" >
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="maximum" >
        <double>180.000000000000000</double>
       </property>
       <property name="value" >
        <double>18.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QCheckBox" name="WithActivity" >
       <property name="text" >
        <string>With Activity</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Activity File      </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="ActivityFile" />
       </item>
      </layout>
     </item>
     <item row="2" column="0" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Activity step     </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QDoubleSpinBox" name="ActivityStep" >
         <property name="singleStep" >
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Date Format</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="dateFormat" >
         <item>
          <property name="text" >
           <string>Select date format</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>YYYY-MM-DD</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>MM/DD/YYYY</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" >
      <widget class="QPushButton" name="Browse" >
       <property name="text" >
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>101</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
